src/webui/www/public/download.html

```diff
@@ -26,6 +26,10 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
     <input type="text" id="cookie" name="cookie" style="width: 16em;"/>
 </div>
 <div class="formRow">
+    <label for="rename" class="leftLabelLarge">QBT_TR(Rename torrent)QBT_TR[CONTEXT=HttpServer]</label>
+	<input type="text" name="rename" style="width: 16em;"/>
```

Comment:
Wrong indent.

Reply:
fixed, also added another commit to convert tab into whitespaces.